u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java

```diff
@@ -23,6 +23,10 @@ public Algorithm getAlgorithm() {
     return algorithm;
   }
 
+  public boolean isEmpty() {
```

Comment:
This change is split out from a bigger change (trying to make smaller pull requests).  I will use this method there (or remove it).